### PR TITLE
Feature/GitHub issues

### PR DESCRIPTION
**Pull request by logan-anderson**
Adds an API that submits GitHub issues to a given repo

### CODE REVIEW

**Comment by DirtyF** [comment]
I like this idea, how does it work, is it testable yet?

**Comment by logan-anderson** [comment]
> I like this idea, how does it work, is it testable yet?

@DirtyF I can update the readme but here are the steps
1. [make a personal GitHub access token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)
2. add new env vars
```env
USERNAME_ISSUES=<your github username>
GITHUB_ACCESS_TOKEN=<Your personal access token>
REPO_ISSUES=< the repo where you want the feedback posted >
```
3. run server
```bash
git checkout feature/github-issues
yarn
yarn dev
```
4. submit feedback in the form on the bottom of the page

**Comment by DirtyF** [comment]
@scottgallant README amended with instructions: https://github.com/tinacms/tina-starter-alpaca/blob/a0574111cb4c6f8cff5957449c563512e3ec7aed/README.md#gathering-feedback

File: components/post-feedback/index.js [hunk 1]
```diff
@@ -1,21 +1,45 @@
...
-    { icon: "icon-happy", value: "happy" },
+    { icon: "icon-confused", value: "Confused😕" },
+    { icon: "icon-neutral", value: "Neutral😐" },
+    { icon: "icon-happy", value: "Happy😄" },
   ]
+  const emojiMap = {
+    "Confused😕": ":confused:",
+    "Neutral😐": ":neutral_face:",
+    "Happy😄": ":smile:",
+  }
```
Comment (DirtyF) [inline components/post-feedback/index.js:21]:
```suggestion
    { icon: "icon-confused", value: "😕 Confused" },
    { icon: "icon-neutral", value: "😐 Neutral" },
    { icon: "icon-happy", value: "😄 Happy" },
  ]
  const emojiMap = {
    "😕 Confused ": ":confused:",
    "😐 Neutral": ":neutral_face:",
    "😄 Happy": ":smile:",
  }
```

Comment (DirtyF) [reply components/post-feedback/index.js:21]: Can we insert spaces here?

**Pull request merged**